torch/_library/utils.py

```diff
@@ -49,3 +51,43 @@ def parse_namespace(qualname: str) -> Tuple[str, str]:
             f"of a namespace and a name, e.g. aten::sin"
         )
     return splits[0], splits[1]
+
+
+def lookup_op(qualname: str) -> torch._ops.OpOverloadPacket:
+    namespace, name = parse_namespace(qualname)
+    if "." in name:
+        name, overload = name.split(".")
+    else:
+        overload = "default"
+    ns = getattr(torch.ops, namespace)
+    packet = getattr(ns, name)
+    return getattr(packet, overload)
+
+
+def is_functional_schema(schema: Any) -> bool:
```

Comment:
We should just use https://github.com/pytorch/pytorch/blob/fd209543d5dfced3146d64df3af6361fc61fb74a/torch/fx/operator_schemas.py#L114 ?